modules/API/API.js

```diff
@@ -58,6 +72,26 @@ function initCommands() {
 
         return false;
     });
+    transport.on('request', ({ data, name }, callback) => {
+        switch (name) {
+        case 'is-audio-muted':
+            callback(APP.conference.audioMuted);
+            break;
+        case 'is-video-muted':
+            callback(APP.conference.videoMuted);
+            break;
+        case 'is-audio-available':
+            callback(audioAvailable);
+            break;
+        case 'is-video-available':
+            callback(videoAvailable);
+            break;
+        default:
+            return false;
+        }
+
+        return true;
```

Comment:
true means that the message is processed and false means that it is not processed. If a message is not processed we are caching the message. You can see the logic in Transport.emit.